packages/api/test/e2e/rx-queries.spec.ts

```diff
@@ -41,11 +42,11 @@ describe.skip('e2e queries', () => {
     api.rpc.chain
       .getHeader()
       .pipe(
-        switchMap((header) =>
-          api.query.system.events.at(header.hash)
+        switchMap(({ hash }: any) =>
```

Comment:
`: Header`

Reply:
With:
```
switchMap(({ hash }: Header) =>
  api.query.system.events.at(hash)
)
```
I'm getting 
`Argument of type 'OperatorFunction<Header, Codec>' is not assignable to parameter of type 'OperatorFunction<Codec, Codec>'.`
` Type 'Codec' is missing the following properties from type 'Header': blockNumber, digest, extrinsicsRoot, hash, and 20 more.`

What am I missing here?